media-library-client/src/app/private-container/pages/home/video-gallery/video-tiles/video-player/video-player-controls.tsx

```diff
@@ -0,0 +1,231 @@
+		},
+		'& .MuiSlider-valueLabel': {
+			lineHeight: 1.2,
+			fontSize: 12,
+			background: 'unset',
+			padding: 0,
+			width: 32,
+			height: 32,
+			borderRadius: '50% 50% 50% 0',
+			backgroundColor: '#52af77',
+			transformOrigin: 'bottom left',
+			transform: 'translate(50%, -100%) rotate(-45deg) scale(0)',
+			'&:before': { display: 'none' },
+			'&.MuiSlider-valueLabelOpen': {
+				transform: 'translate(50%, -100%) rotate(-45deg) scale(1)',
+			},
+			'& > *': {
+				transform: 'rotate(45deg)',
+			},
+		},
+	});
+
+	function handlePopOver(event: React.MouseEvent<HTMLButtonElement, MouseEvent>): void {
+		if (event && event.currentTarget) {
+			setAnchorEl(event.currentTarget);
+		}
+	}
+
+	return (
+		<div className="controls__div">
+			<Grid
+				container
+				direction="row"
+				alignItems="center"
+				justifyContent="space-between"
+				style={{ padding: 16 }}
+			>
+				<Grid item xs={12}>
+					<PrettoSlider
+						min={0}
+						max={100}
+						value={played * 100}
+						onChange={onSeek}
+						onMouseDown={onSeekMouseDown}
+						onChangeCommitted={onSeekMouseUp}
+						valueLabelDisplay="auto"
+						// aria-label="custom thumb label"
+						components={{
+							ValueLabel: ValueLabelComponent,
+						}}
+					/>
+					<Grid container direction="row" justifyContent="space-between">
+						<Typography style={{ color: 'white' }}>{playedTime}</Typography>
+						<Typography style={{ color: 'white' }}>{fullMovieTime}</Typography>
+					</Grid>
+				</Grid>
+
+				<Grid item xs={12}>
+					<Grid container alignItems="center" direction="row">
+						<IconButtonWrapper id={'rewind'} icon={IconSelector.FastRewind} onClick={rewind} />
+                        <div>
+                        {playing ? (
+								<IconButtonWrapper id={'play'} icon={IconSelector.Pause} onClick={playandpause} />
+							) : (
+								<IconButtonWrapper id={'pause'} icon={IconSelector.PlayArrow} onClick={playandpause} />
+							)}
+                        </div>
+                        
+                        <IconButtonWrapper id={'forward'} icon={IconSelector.FastForward} onClick={fastForward} />
+                        <div>
+                        {muted ? (
+								<IconButtonWrapper id={'volume-off'} icon={IconSelector.VolumeUp} onClick={muting} />
+							) : (
+								<IconButtonWrapper id={'volume-up'} icon={IconSelector.VolumeOff} onClick={muting} />
+							)}
+                        </div>
+
+						{/* <IconButton className="controls__icons" aria-label="reqind" onClick={playandpause}>
+							{playing ? (
+								<PauseSharp fontSize="large" style={{ color: 'white' }} />
+							) : (
+								<PlayArrowSharp fontSize="large" style={{ color: 'white' }} />
+							)}
+						</IconButton> */}
+
+						<Typography style={{ color: '#fff', paddingTop: '5px' }}>
+							{Math.ceil(volume * 100)}
+						</Typography>
+						<Slider
+							min={0}
+							max={100}
+							value={volume * 100} // Assuming volume is between 0 and 1

```

Comment:
Did we test for potential keyboard inputs rather than just using the mouse to interact with the slider?

Reply:
Not yet - Need to investigate Potential Events for Keyboard Actions to add